src/components/Stake/StakeGauge.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="gauge">
+    <div class="explanation">
+      <div class="title">SharedStaked vEth2</div>
+      <div class="content">
+        Total of {{ ethDeposited }}Eth has been staked to SharedStake so far.
+        <br />
+        32100Eth is required (for 1000 validators) for the third deposit to ETH2 event
```

Comment:
There's actually nothing more than 32 eth needed for the 3rd deposit. The tardiness has been due to figuring out more cloud providers and regions and issues with AWS. 

- Can we therefore remove the comment about the 3rd deposit? 
- Can we change `Eth` to `ETH`? 
- Can we have the base value of 32100 be dynamic? (from teh contract it's costPerValidator * numValidators)